core/src/main/java/cz/xtf/core/openshift/OpenShifts.java

```diff
@@ -117,7 +117,7 @@ public static OpenShiftBinary adminBinary(String namespace) {
                 OpenShiftConfig.adminKubeconfig(), namespace);
     }
 
-    private static OpenShiftBinary getBinary(String token, String username, String password, String kubeconfig,
```

Comment:
This is not necessary after deleting dependency between HelmBinary and OpenShiftBinary?

Reply:
I have made quite some changes, would you have another look if this thread can be resolved?

Reply:
I see this method was resolved from OpenShifts. Resolving.